{"name": "Пользователь может осуществить поиск (failed)", "status": "failed", "statusDetails": {"message": "AssertionError: МТС - связь и экосистема цифровых сервисов failed is not presented in result number 0", "trace": "browser = <selenium.webdriver.remote.webdriver.WebDriver (session=\"853f501017d65022dedf493e6bbdb6db\")>\npage = <pages.main_page.MainPage object at 0x000002A3DC5692E0>\n\n    @allure.title('Пользователь может осуществить поиск (failed)')\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_guest_can_go_to_checkout_failed(browser, page):\n        with allure.step('Ввод запроса и нажатие кнопки \"Поиск в Google\"'):\n            page.enter_search_query(\"МТС\")\n            page.click_search_button()\n        with allure.step('Проверка, что первый результат соответствует ожиданиям'):\n            results_page = ResultsPage(browser, links.MAIN_PAGE)\n>           results_page.should_has_result_with_correct_title(0, \"МТС - связь и экосистема цифровых сервисов failed\")\n\ntests\\test_main_page.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.results_page.ResultsPage object at 0x000002A3DC637BE0>\nresult_number = 0\nresult_title = 'МТС - связь и экосистема цифровых сервисов failed'\n\n    def should_has_result_with_correct_title(self, result_number, result_title):\n>       assert self.browser.find_elements(*ResultPageLocators.RESULTS)[result_number].text.find(result_title) != -1, f\"{result_title} is not presented in result number {result_number}\"\nE       AssertionError: МТС - связь и экосистема цифровых сервисов failed is not presented in result number 0\n\npages\\results_page.py:7: AssertionError"}, "steps": [{"name": "Ввод запроса и нажатие кнопки \"Поиск в Google\"", "status": "passed", "start": 1608194044769, "stop": 1608194048003}, {"name": "Проверка, что первый результат соответствует ожиданиям", "status": "failed", "statusDetails": {"message": "AssertionError: МТС - связь и экосистема цифровых сервисов failed is not presented in result number 0\n", "trace": "  File \"C:\\example_ui_tests\\tests\\test_main_page.py\", line 36, in test_guest_can_go_to_checkout_failed\n    results_page.should_has_result_with_correct_title(0, \"МТС - связь и экосистема цифровых сервисов failed\")\n  File \"C:\\example_ui_tests\\pages\\results_page.py\", line 7, in should_has_result_with_correct_title\n    assert self.browser.find_elements(*ResultPageLocators.RESULTS)[result_number].text.find(result_title) != -1, f\"{result_title} is not presented in result number {result_number}\"\n"}, "start": 1608194048003, "stop": 1608194048978}], "start": 1608194044768, "stop": 1608194048980, "uuid": "ae225a83-871a-4f92-aa62-838b84b06a2a", "historyId": "7aaa128148c91fa43eff8f01ef4bf52b", "testCaseId": "b2550cb1d36fac0cadf4353820eb19c9", "fullName": "tests.test_main_page#test_guest_can_go_to_checkout_failed", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_main_page"}, {"name": "host", "value": "DESKTOP-4VE46VG"}, {"name": "thread", "value": "16656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_main_page"}]}